### PR TITLE
nf-test-init-config-pipeline-minimal-tests-snaps

### PR DESCRIPTION
**Pull request by sateeshperi**
## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/ampliseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/ampliseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by d4straub** [comment]
Interesting, seems to check for file existence and file md5sums for test results?

**Comment by sateeshperi** [comment]
that's correct. I try to snapshot (named md5sum json) everything in the output folder for each test profile and do file exists checks for files that can't generate a consistent snapshot. It is still POC but you can give it a try by installing [nf-test](https://code.askimed.com/nf-test/installation/) and running `nf-test test` from the pipeline repo.

File: tests/minimal_doubleprimers.nf.test [hunk 1]
```diff
@@ -0,0 +1,43 @@
+nextflow_pipeline {
+
+    name "Test Workflow main.nf"
+    script "main.nf"
+    tag "nf-core/ampliseq"
+
+    test("Minimal DoublePrimers") {
+
+        options "-profile test_doubleprimers,docker"
```
[the remark this answers is not in the document]
Comment (sateeshperi) [reply tests/minimal_doubleprimers.nf.test]: updated

**Comment by Emiller88** [comment]
These can probably be cleaned up with a rebase 
image

**Pull request closed without merging**